src/me/mrCookieSlime/ExoticGarden/PlantsListener.java

```diff
@@ -220,6 +222,15 @@ public void onHarvest(BlockBreakEvent e) {
 
 	@EventHandler(priority = EventPriority.HIGHEST, ignoreCancelled = true)
 	public void onDecay(LeavesDecayEvent e) {
+		SlimefunItem sfItem = BlockStorage.check(e.getBlock());
```

Comment:
This method gets the item's id, then looks up the item.
If you are only using this for id comparison, then you should rather only use the method that returns the ID.